src/Microsoft.SqlTools.ServiceLayer/LanguageServices/AutoCompleteHelper.cs

```diff
@@ -754,6 +754,14 @@ public static CompletionItem[] ExpandSqlStarExpression(ScriptDocumentInfo script
                 starObjectIdentifier = (SqlObjectIdentifier)selectStarExpression.Children.ElementAt(0);
             }
 
+            /*
+            Returning no suggestions when the bound tables are null. 
+            This happens when there are no existing connections for the script. 
+            */ 
+            if(selectStarExpression.BoundTables == null){

```

Comment:
the coding convention in this project is following .Net Framework guidelines, so this code block should be written as below (with spacing and brace on separate line):

```
if (selectStarExpression.BoundTables == null)
{
     return null;
}
```

Reply:
(if you format the doc it should fix all this for you)